sdk/core/core-arm/package.json

```diff
@@ -87,7 +87,11 @@
     "test": "npm run build && npm run unit-test && npm run integration-test",
     "unit-test": "npm run unit-test:node && npm run unit-test:browser",
     "unit-test:browser": "echo skipped",
-    "unit-test:node": "nyc mocha --require ts-node/register --timeout 50000 --reporter ../../../common/tools/mocha-multi-reporter.js --colors \"test/**/*.ts\"",
+    "unit-test:node": "mocha --require ts-node/register --timeout 50000 --reporter ../../../common/tools/mocha-multi-reporter.js --colors \"test/**/*.ts\"",
+    "cover:unit:browser": "nyc npm run unit-test:browser",
+    "cover:unit:node": "nyc npm run unit-test:node",
+    "cover:unit": "npm run cover:unit:node && npm run cover:unit:browser",
+    "cover:report": "nyc report --reporter=lcov --reporter=text",
```

Comment:
How does this work?

Reply:
i am not sure what do you mean? these are normal npm scripts.

Reply:
So, does this command take `.nyc_output` as the input and generate the coverage?

We already have the `.nycrc` that has the config to generate coverage in some formats. 
I don't think we need another command.

Reply:
> We already have the .nycrc that has the config to generate coverage in some formats.
> I don't think we need another command.

I do not see `.nycrc` here. To make sure I understand your comment, you suggest to use some existing format for the coverage report rather than `--reporter=lcov --reporter=text`?

Reply:
Maybe we missed adding it to the core-arm package.
Usually, if you look at the SDKs like storage or service-bus, you'll find the `.nycrc` file which is the configuration that is picked up by the `nyc` command when the tests are executed with `nyc mocha`.
![image](https://user-images.githubusercontent.com/10452642/88108575-f3266c00-cb5d-11ea-888c-d020df5a1dbe.png)
The config is not just for the reports but also for the files to be included/excluded in the coverage, to pick up sourcemaps.


> To make sure I understand your comment, you suggest to use some existing format for the coverage report rather than --reporter=lcov --reporter=text?

Yes, the reports are already generated when the tests are executed with `nyc mocha`. So, I feel there is no need for this script/command. If some SDKs are missing the config, we need to make sure to add them.

Reply:
Yes I know they exist elsewhere. However adding `.nycrc` to `core-arm` is out of the scope of this PR I think if this is what you are suggesting. I will remove this script for now and we can discuss how to standardize coverage later in another issue.

Reply:
Sure, you can log an issue perhaps!

Reply:
I actually found #5342 to be pretty relevant. @jeremymeng do you still see this performance degradation?

Reply:
nyc always adds overhead without providing real benefits when we are not looking at the results but we decided at some point that it is good for nightly to have the code coverage reports available from the build pipelines.  At the same time we also think it's not too troublesome to remove `nyc` when running integration tests locally. So #5342 can be closed now.